packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -133,6 +133,7 @@ export default class GoogleCalendarService implements Calendar {
           calendarId: selectedCalendar,
           requestBody: payload,
           conferenceDataVersion: 1,
+          sendUpdates: "none",
```

Comment:
https://developers.google.com/calendar/api/v3/reference/events/insert#parameters

Reply:
<img width="596" alt="image" src="https://user-images.githubusercontent.com/4461358/227667125-6cf72945-b33e-4310-815d-57be349f15f1.png">

Reply:
@joeauyeung @zomars @emrysal are we really fine with these sideeffects?

Reply:
Yes, these warnings exists for typical applications "Create a booking" - it's not for booking software that sends booking invites on its own like we do 🙏